demo/b2share_demo/helpers.py

```diff
@@ -315,7 +315,8 @@ def download_v1_data(token, target_dir, limit=None, verbose=False):
         if len(recs) == 0:
             return # no more records
         for record in recs:
-            if record.get('files') == 'RESTRICTED':
+            if (record.get('files') == 'RESTRICTED' or 
+                not(record.get('open_access')) ):
```

Comment:
I disagree with this change, we should at least try to make the record and migrate the files afterward, in other ways